docs/platform/developing/building-all.md

```diff
@@ -57,203 +57,58 @@ To clone the latest sources of all projects over SSH:
    Initialize the repository by executing the following command:
 
    ```
-   $ repo init -u ssh://<Username>@review.tizen.org:29418/scm/manifest -b <tizen branch> -m ${profile}_${repository}.xml
+   $ repo init -u ssh://<Username>@review.tizen.org:29418/scm/manifest -b <tizen branch> -m <profile>_<repository>.xml
    ```
+   > **Note**
+   >
+   > If you encounter some problems using repo tools, please check [Setting Up the Repo Tool](./setting-up.md#setting-up-the-repo-tool).
 
-    `<tizen branch>` is `tizen` for Tizen 5.0, `tizen_4.0` for Tizen 4.0 and `tizen_3.0` for Tizen 3.0.
+    `<tizen branch>` is `tizen` for Tizen latest version and others `tizen_<version> branch` are `tizen <version>` for Tizen specific version include old versions.
```

Comment:
@mijin85cho, Thank you for sharing the information. Kindly check if the following changes looks fine?

`<tizen branch>` is `tizen` for Tizen latest version, and other `tizen_<version> branches` are `tizen <version>` for specific Tizen version, including older versions.

Reply:
@sagarkedari Seems much more appropriate, and it would be better if the location of \` is changed as `tizen_<version>` is the word used, not `tizen_<version> branches`.

`<tizen branch>` is `tizen` for Tizen latest version, and other `tizen_<version>` branches are `tizen <version>` for specific Tizen version, including older versions.

Reply:
@mijin85cho, thank you for checking and suggesting the change. I will get this changed per your suggestion.